client/src/main/java/io/pravega/client/stream/impl/StreamCutImpl.java

```diff
@@ -64,16 +69,28 @@ public Stream getStream() {
     public StreamCutInternal asImpl() {
         return this;
     }
-    
+
     @Override
     public String toString() {
-        return stream.getScopedName() + ":"
+        return stream.getScopedName() + ":" + TO_STRING_VERSION + ":"
```

Comment:
Let's not overload toString. This really should be for debugging. If we need to create a method that is going to be versioned for serialization, I think it should called something else.

Reply:
done.